storage/remote/otlptranslator/prometheus/normalize_name.go

```diff
@@ -236,8 +244,11 @@ func removeSuffix(tokens []string, suffix string) []string {
 }
 
 // Clean up specified string so it's Prometheus compliant
-func CleanUpString(s string) string {
-	return strings.Join(strings.FieldsFunc(s, func(r rune) bool { return !unicode.IsLetter(r) && !unicode.IsDigit(r) }), "_")
+func CleanUpString(s string, allowUTF8 bool) string {
+	if !allowUTF8 {
+		return strings.Join(strings.FieldsFunc(s, func(r rune) bool { return !unicode.IsLetter(r) && !unicode.IsDigit(r) }), "_")
+	}
+	return s
 }
```

Comment:
I don't think this function should change, since when `allowUTF8 == true` it's a no-op. Just don't call it in that case. Also saves you from having to write tests proving it doesn't change the input when `allowUTF8 == true`.